src/app/MessageDef.h

```diff
@@ -0,0 +1,1854 @@
+/**
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2016-2017 Nest Labs, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+/**
+ *    @file
+ *      This file defines CHIP interaction model message parsers and encoders
+ *      Interaction model profile.
+ *
+ */
+
+#ifndef _CHIP_INTERACTION_MODEL_MESSAGE_DEF_H
```

Comment:
#pragma once, please in all these headers

Reply:
Done, thanks